packages/dropdown/dropdown.scss

```diff
@@ -18,6 +18,8 @@ $side-padding: $component-spacing--small;
 $bottom-padding: $component-spacing--small;
 $bottom-margin: $component-spacing--large;
 
+$dropdown-total-height: rem(68px);
```

Comment:
How about `$label-height + $value-height + $bottom-padding`? Makes it super clear and updates automagically 😎

Reply:
Then it will cover the bottom line, but that kinda looks better. So I've updated it.